app/DTO/UserRights.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\DTO;
+
+use App\Contracts\AbstractAlbum;
+use App\Models\User;
+use App\Policies\AlbumPolicy;
+use App\Policies\UserPolicy;
+use Illuminate\Support\Facades\Gate;
+
+/**
+ * Data Transfer Object (DTO) to transmit the rights of a User.
+ */
+class UserRights extends ArrayableDTO
```

Comment:
This DTO doesn't make sense to me (again). Rights are something which the currently authenticated user has on some other object "under investigation". This is true for `AlbumRights`, `PhotoRights` etc. They contain the effective rights of the currently authenticated user on that album or photo.

Following the same structure, `UserRights` has summarize the effective rights which the currently authenticated user has on some other user. (Remark: This other user can also be identical the authenticated user. in that case this object would represent the rights of the user on its own account.) In this kind of systematics, this object has to represent rights of a user on a user, for example: `CAN_CHANGE_PASSWORD`. This would make sense, e.g. can user X change the password of user Y. This right would be granted for administrators and if the authenticated user is identical to the user under investigation.

However, here we have things like "CAN_UPLOAD_ROOT". This is not a right of a user on another user.